ondemand.osc.edu/apps/myjobs/templates/Basic_LAMMPS_Parallel_Job_Pitzer-Exp/slurm_script.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#SBATCH -J ondemand/sys/myjobs/basic_lammps_parallel
+#SBATCH --time=00:30:00
+#SBATCH --nodes=2
+#SBATCH --ntasks-per-node=48
```

Comment:
Per my other comment, remove this restriction to ensure this script can work on both old and new Pitzer.